documentation-website/Writerside/topics/DAO-Entity-definition.topic

```diff
@@ -230,10 +230,25 @@
                 Although it is still possible to insert or update values with negative integers via DAO, this approach assures
                 a cleaner business logic.
             </p>
-            <note>
-                Note that these transformations will occur with every access to a field. Therefore, to maintain performance
-                efficiency, it is advisable to avoid heavy transformations in this context.
-            </note>
+        </chapter>
+
+
+        <chapter title="Memoized transformations" id="memoized-transformations">
+            <p>
+                If you have a computationally expensive transformation function,
```

Comment:
Is "computationally expensive" a common expression ? For added clarity, I would suggest something like "If your transformation function involves complex operations that impact performance..."